Modules/posixmodule.c

```diff
@@ -8588,6 +8588,49 @@ os_pidfd_open_impl(PyObject *module, pid_t pid, unsigned int flags)
 #endif
 
 
+#if defined(__linux__) && defined(HAVE_SETNS)
+/*[clinic input]
+os.setns
+  fd: fildes
+  nstype: int = 0
+
+Allows the calling thread to move into different namespaces.
```

Comment:
Please use imperative mood for both docstrings. Also, you could consider adding short docstrings for each parameter.